packages/venia-ui/lib/RootComponents/Category/NoProductsFound/noProductsFound.js

```diff
@@ -9,17 +9,14 @@ import { mergeClasses } from '../../../classify';
 import noProductsFound from './noProductsFound.png';
 import defaultClasses from './noProductsFound.css';
 
-// TODO: get categoryUrlSuffix from graphql storeOptions when it is ready
-const categoryUrlSuffix = '.html';
-
 const NoProductsFound = props => {
     const { recommendedCategories } = useNoProductsFound(props);
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const categoryItems = useMemo(() => {
         return recommendedCategories.map(category => {
             const uri = resourceUrl(
-                `/${category.url_path}${categoryUrlSuffix}`
+                `/${category.url_path}${category.url_suffix}`
```

Comment:
Once you switch to using the `url_suffix` field on the category response this should actually work. I don't think this works as is because the catalog reducer does not know what to do with the `storeConfig` section of the response data.